src/Surfnet/Migrations/Version20210610131957.php

```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
+
+namespace Surfnet\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use function json_decode;
+use function json_encode;
+
+/**
+ * This migration removes sensitive data (vetting type) from the event stream
+ * This data ended up by accident, this migration loads the targeted events and
+ * removes the sensitive data via JSON encoding/decoding the payload of the event.
+ */
+final class Version20210610131957 extends AbstractMigration
+{
+    private static $select = <<<SQL
+        SELECT uuid, playhead, payload
+        FROM event_stream
+        WHERE (type = 'Surfnet.Stepup.Identity.Event.SecondFactorVettedEvent'
+            OR type = 'Surfnet.Stepup.Identity.Event.SecondFactorVettedWithoutTokenProofOfPossession')
+        AND recorded_on > '2021-04-11'
```

Comment:
Should we check for the presence of document_number, instead of a recorded_on? The date is very SURF-specific, and a document number should never be present in the event_stream.

Reply:
In the line below it's checking for the `vetting_type` which is the element which holds the 'document_number'. That element should be removed completely, also for other types of VettingTypes. So that's why the `vetting_type` is being used. The `recorded_on` is used to only migrate events after the date were the bug was introduced to prevent iterating unnecessary all events. In the original change a between was used on the `recorded_on` data and I did remove that in order to support other users who could potentially run the migration later in time.